composer.json

```diff
@@ -219,6 +219,7 @@
     "config": {
         "process-timeout": 0,
         "allow-plugins": {
+            "php-http/discovery": false,
```

Comment:
This is not about this PR, but let's also fix EasyApiToken tests.

Reply:
Done, plugin disabled in EasyApiToken too

Reply:
Thanks